classes/ImageManager.php

```diff
@@ -628,6 +678,18 @@ public static function write($type, $resource, $filename)
             case 'png':
                 $quality = ($psPngQuality === false ? 7 : $psPngQuality);
                 $success = imagepng($resource, $filename, (int) $quality);
+                if (Configuration::get('PS_IMAGEOPT_PNGQUANT')) {
+                    $pngquant_ext = pathinfo($filename, PATHINFO_EXTENSION) == 'png' ? '--ext=.png' : '--ext=';
+                    exec('/usr/bin/pngquant -f ' . $pngquant_ext . ' ' . escapeshellarg($filename));
+                }
+                if (Configuration::get('PS_IMAGEOPT_OPTIPNG')) {
+                    exec('/usr/bin/optipng -o7 -q ' . escapeshellarg($filename));
```

Comment:
Seems very specific, I'm afraid that in another binary system is elsewhere.

Reply:
Is it enough, if I remove the path, and change the comment of the from `Efficient bandwidth and disk usage optimization. Package pngquant must be installed in /usr/bin.` to `Efficient bandwidth and disk usage optimization. Package pngquant must be available in the path for the apache user.`  This would be much less specific. 
Going further than this would probably imply to test on different configurations.

Reply:
@PululuK I made the changes. Is this OK to you?